cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -6,24 +6,34 @@
 
    Description:
 
-   Creates markup for a Card molecule, which displays an icon, some text,
-   and a link.
+   Creates markup for a Card molecule, which displays some text and a link,
+   with an optional heading and icon.
 
    value.text:                  Card body text.
 
    value.link_text:             Card link text.
 
    value.link_url:              Card link URL.
 
+   value.heading:               Card heading (optional).
+
+   value.icon:                  Card icon (optional).
+
    ========================================================================== #}
 
 {% macro render( value ) %}
 {# We want to make the whole card clickable. #}
 <article class="m-card m-card__featured">
     <a href="{{ value.link_url }}">
         <h2 class="m-card_heading">
-            <span class="m-card_icon">{{ svg_icon( value.icon ) }}</span>
-            <span class="u-visually-hidden">{{ value.link_text }}</span>
+            {% if value.icon -%}
+                <span class="m-card_icon">{{ svg_icon( value.icon ) }}</span>
+            {%- endif %}
+            {% if value.heading -%}
```

Comment:
Hmm this makes heading optional, but in the Wagtail admin it's requiring me to put a heading in for the cards?

Reply:
The card _template_ supports cards with and without headings (the current home page vs. the new), whereas the card _block_ I've prototyped here specifically for the home page does require one.